lib/features/give/pages/giving_page.dart

```diff
@@ -67,14 +67,12 @@ class _GivingPageState extends State<GivingPage> {
     final afterGivingRedirection =
         context.read<GiveBloc>().state.afterGivingRedirection;
 
-    context.read<AuthCubit>().state.user.isUsUser
-        ? context.pop()
-        : context.goNamed(
-            Pages.home.name,
-            queryParameters: {
-              'given': 'true',
-            },
-          );
+    context.goNamed(
+      Pages.home.name,
```

Comment:
Is this truly the correct route? Where is it supposed to go? ProfileSelectionScreen or the parent home page? Or somewhere else?

Reply:
uh this comment is u=outdated, but for some reason not marked so, im confused

Reply:
all changes on the givign_page.dart have been reverted including this, and in the ParentGivingPage its context.pop